Xamarin.Forms.Platform.Tizen/Renderers/EditorRenderer.cs

```diff
@@ -153,5 +154,10 @@ string MaxLengthFilter(ElmSharp.Entry entry, string s)
 
 			return null;
 		}
+
+		void UpdateIsReadOnly()
+		{
+			Control.AllowFocus(!Element.IsReadOnly);
```

Comment:
I tested this on Tizen and readonly didn't seem to have any effect
There's an *IsEditable* property though.  I'm guessing that's the field to use

Reply:
I will try that also, thank you!

Reply:
uhmm.. I'll look further inside.

Reply:
```Control.IsEditable = !Element.IsReadOnly;``` works fine.